projects/packages/forms/src/contact-form/class-feedback.php

```diff
@@ -625,7 +625,10 @@ public function get_compiled_fields( $context = 'default', $array_shape = 'all'
 					$compiled_fields[ $field->get_key() ] = $field->get_render_value( $context );
 					break;
 				case 'label-value':
-						$compiled_fields[ $field->get_label( $context, $count_field_labels[ $label ] ) ] = $field->get_render_value( $context );
+					$compiled_fields[ $field->get_label( $context, $count_field_labels[ $label ] ) ] = $field->get_render_value( $context );
+					break;
+				case 'id-value':
+					$compiled_fields[ $field->get_form_field_id() ] = $field->get_render_value( $context );
```

Comment:
Lets also add a test for this new shape :)

Reply:
Added, but had to use custom expected array value since dataprovider doesn't allow to pass down any arguments 🙃